src/databricks/labs/ucx/framework/crawlers.py

```diff
@@ -147,15 +147,15 @@ def __init__(self):
             msg = "Not in the Databricks Runtime"
             raise RuntimeError(msg)
 
-        self._spark = SparkSession.builder.getOrCreate()
+        self.spark = SparkSession.builder.getOrCreate()
```

Comment:
How many times do i have to repeat that you're not allowed to make any change like that to `RuntimeBackend`? Invoke `SparkSession.builder.getOrCreate()` + runtime check from `TableSizes`.

Reply:
Fixed